### PR TITLE
Update enterprise-search-audit.md

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) updates the S3 links for dashboard screenshots.

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 1]
```diff
@@ -32,28 +32,28 @@ To install the app:
...
-### Search Overview
+### Enterprise Search Audit - Search Overview
 The Enterprise Search Audit - Search Overview dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of query, and other query characteristics run by users.
 
 Use this dashboard to:
 * To quickly monitor searches run in your account, including by search type.
 * To get insights into query statistics, time ranges, and query failure rate.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-SearchOverview.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Search-Overview.png')} alt="test" />
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:42]: I'd change the alt text to match the images for all four of these dashboards images

```suggestion
<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Search-Overview.png')} alt="Enterprise Search Audit - Search Overview" />
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 2]
```diff
@@ -32,28 +32,28 @@ To install the app:
...
+### Enterprise Search Audit - Data Tier Usage Over Time
 The Enterprise Search Audit - Data Tier Usage Over Time dashboard provides detailed insights into search usage by analytics tier. It displays the total number of searches and the types of queries run by users.
 
 Use this dashboard to:
 
 * To drill-down on search usage by different analytics tiers.
 * Quickly Identify trends and usage patterns within each analytic tier.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-DataTierUsageOverTime.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Data-Tier-Usage-Over-Time.png')} alt="test" />
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:53]:
```suggestion
<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Data-Tier-Usage-Over-Time.png')} alt="Enterprise Search Audit - Data Tier Usage Over Time" />
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 3]
```diff
@@ -62,20 +62,20 @@ Use this dashboard to:
 * To view the distribution of query time range.
 * Identify top users for each query type in every tier.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-UserInsights.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-User-Insights.png')} alt="test" />
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:65]:
```suggestion
<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-User-Insights.png')} alt="Enterprise Search Audit - User Insights" />
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 4]
```diff
@@ -62,20 +62,20 @@ Use this dashboard to:
...
+### Enterprise Search Audit - Commonly Referenced Data Sources
 The Enterprise Search Audit - Commonly Referenced Data Sources dashboard provides insights into the key or most popular data sources and metadata referenced in users’ searches. The dashboard helps you understand what your users most commonly search for in your organization.
 
 Use this dashboard to:
 
 * Identify the most commonly referenced partitions (indexes) and scheduled views to gauge user adoption and awareness of these options.
 * Identify the most commonly referenced source categories and other metadata in search queries to understand the most commonly used/relied upon data sources within your organization.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-CommonlyReferencedDataSources.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Commonly-Referenced-Data-Sources.png')} alt="test" />
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:75]:
```suggestion
<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Commonly-Referenced-Data-Sources.png')} alt="Enterprise Search Audit - Commonly Referenced Data Sources" />
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 5]
```diff
@@ -84,4 +84,4 @@ Use this dashboard to:
 * Identify which queries span large time ranges and may be costly in terms of performance.
 * To identify the searches that do not apply any optimization techniques and could benefit from optimization.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-QueriesCharacteristicsAndOpportunitiestoImprove.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Queries-Characteristics-and-Opportunities-to-Improve.png')} alt="test" />
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:87]:
```suggestion
<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Queries-Characteristics-and-Opportunities-to-Improve.png')} alt="Enterprise Search Audit - Queries Characteristics and Opportunities to Improve" />
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 6]
```diff
@@ -32,28 +32,28 @@ To install the app:
 This page has information about the pre-built dashboards for Enterprise Search Audit App and a description of each of the app dashboards.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:32]:
```suggestion
This page has information about the pre-built dashboards for the Enterprise Search Audit App and a description of each of the app dashboards.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 7]
```diff
@@ -32,28 +32,28 @@ To install the app:
...
 
 
-### Search Overview
+### Enterprise Search Audit - Search Overview
 The Enterprise Search Audit - Search Overview dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of query, and other query characteristics run by users.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:36]:
```suggestion
The **Enterprise Search Audit - Search Overview** dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of queries, and other query characteristics run by users.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 8]
```diff
@@ -32,28 +32,28 @@ To install the app:
...
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Search-Overview.png')} alt="test" />
 
 
-### Data Tier Usage Over Time
+### Enterprise Search Audit - Data Tier Usage Over Time
 The Enterprise Search Audit - Data Tier Usage Over Time dashboard provides detailed insights into search usage by analytics tier. It displays the total number of searches and the types of queries run by users.
 
 Use this dashboard to:
 
 * To drill-down on search usage by different analytics tiers.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:50]:
```suggestion
* To drill down on search usage by different analytics tiers.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 9]
```diff
@@ -32,28 +32,28 @@ To install the app:
...
 * Quickly Identify trends and usage patterns within each analytic tier.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:51]:
```suggestion
* To quickly identify trends and usage patterns within each analytic tier.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 10]
```diff
@@ -84,4 +84,4 @@ Use this dashboard to:
 * Identify which queries span large time ranges and may be costly in terms of performance.
 * To identify the searches that do not apply any optimization techniques and could benefit from optimization.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:85]:
```suggestion
* Identify the searches that do not apply any optimization techniques and could benefit from optimization.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md (hunk 9 above)
Comment (JainM6) [reply docs/integrations/sumo-apps/enterprise-search-audit.md:51]: Consistency

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 11]
```diff
@@ -62,20 +62,20 @@ Use this dashboard to:
 * To view the distribution of query time range.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:62]:
```suggestion
* View the distribution of query time range.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 12]
```diff
@@ -62,20 +62,20 @@ Use this dashboard to:
...
 * Identify top users for each query type in every tier.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:63]:
```suggestion
* To identify top users for each query type in every tier.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 13]
```diff
@@ -29,31 +29,31 @@ To install the app:
...
 
 
-### Data Tier Usage Over Time
+### Enterprise Search Audit - Data Tier Usage Over Time
 The Enterprise Search Audit - Data Tier Usage Over Time dashboard provides detailed insights into search usage by analytics tier. It displays the total number of searches and the types of queries run by users.
 
 Use this dashboard to:
 
 * To drill-down on search usage by different analytics tiers.
 * Quickly Identify trends and usage patterns within each analytic tier.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:51]:
```suggestion
* Drill down on search usage by different analytics tiers.
* Quickly Identify trends and usage patterns within each analytic tier.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 14]
```diff
@@ -29,31 +29,31 @@ To install the app:
...
 
 
-### Search Overview
-The Enterprise Search Audit - Search Overview dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of query, and other query characteristics run by users.
+### Enterprise Search Audit - Search Overview
+The **Enterprise Search Audit - Search Overview** dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of queries, and other query characteristics run by users.
 
 Use this dashboard to:
 * To quickly monitor searches run in your account, including by search type.
 * To get insights into query statistics, time ranges, and query failure rate.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:40]:
```suggestion
* Quickly monitor searches run in your account, including by search type.
* Get insights into query statistics, time ranges, and query failure rate.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 15]
```diff
@@ -32,28 +32,28 @@ To install the app:
...
 * To drill-down on search usage by different analytics tiers.
 * Quickly Identify trends and usage patterns within each analytic tier.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-DataTierUsageOverTime.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Data-Tier-Usage-Over-Time.png')} alt="test" />
 
 
-### User Insights
+### Enterprise Search Audit - User Insights
 The Enterprise Search Audit - User Insights dashboard provides a high-level view to understand who are the key users of search in your account. The dashboard panels provide details about search usage of top users in terms of search query count, data scanned and retrieved by tiers.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:57]:
```suggestion
The Enterprise Search Audit - User Insights dashboard provides a high-level view of your account's key search users. The dashboard panels provide information about top users' search usage in terms of search query count, data scanned and retrieved by tiers.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 16]
```diff
@@ -29,59 +29,59 @@ To install the app:
...
-### Data Tier Usage Over Time
+### Enterprise Search Audit - Data Tier Usage Over Time
 The Enterprise Search Audit - Data Tier Usage Over Time dashboard provides detailed insights into search usage by analytics tier. It displays the total number of searches and the types of queries run by users.
 
 Use this dashboard to:
 
-* To drill-down on search usage by different analytics tiers.
-* Quickly Identify trends and usage patterns within each analytic tier.
+* To drill down on search usage by different analytics tiers.
+* To quickly identify trends and usage patterns within each analytic tier.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:51]:
```suggestion
* Drill down on search usage by different analytics tiers.
* Quickly identify trends and usage patterns within each analytic tier.
```

File: docs/integrations/sumo-apps/enterprise-search-audit.md (hunk 15 above)
Comment (JainM6) [reply docs/integrations/sumo-apps/enterprise-search-audit.md]: I'd rephrase this as:
The **Enterprise Search Audit - User Insights** dashboard provides a high-level view of your account's key search users. The dashboard panels provide information about top users' search usage in terms of search query count, data scanned and retrieved by tiers.

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 17]
```diff
@@ -29,59 +29,59 @@ To install the app:
...
+### Enterprise Search Audit - User Insights
+The Enterprise Search Audit - User Insights dashboard provides a high-level view of your account's key search users. The dashboard panels provide information about top users' search usage in terms of search query count, data scanned and retrieved by tiers.
 
 Use this dashboard to:
 
 * Monitor top users by search count, data scanned and retrieved across the account.
-* To view the distribution of query time range.
-* Identify top users for each query type in every tier.
+* View the distribution of query time range.
+* To identify top users for each query type in every tier.
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:63]:
```suggestion
* Identify top users for each query type in every tier.
```

**Review by kimsauce — APPROVED** [review]

File: docs/integrations/sumo-apps/enterprise-search-audit.md [hunk 18]
```diff
@@ -84,4 +84,4 @@ Use this dashboard to:
 * Identify which queries span large time ranges and may be costly in terms of performance.
```
Comment (JainM6) [inline docs/integrations/sumo-apps/enterprise-search-audit.md:84]:
```suggestion
To identify queries that span large time periods and are likely to be performance-intensive.
```
